docs/building.md

```diff
@@ -0,0 +1,29 @@
+## Building, flashing, and debugging
+
+### Build
+[Here](https://github.com/lpn-plant/ms-tpm-20-ref) you can find our fork of [Official TPM 2.0 Reference Implementation (by Microsoft)](https://github.com/microsoft/ms-tpm-20-ref).
+
+The project is developed using STM32CubeIDE so the building process is straightforward.
+
+Just clone it with git...
+```
+git clone --recurse-submodules git@github.com:lpn-plant/ms-tpm-20-ref.git
+```
+
+... and import it using `Import -> Existing Project into Workspace` in `IDE Project Explorer`.
+
+Select `Nucleo-L476RG` project from `ms-tpm-20-ref/Samples/Nucleo-TPM/L476RG` directory.
+
+### Flash and Debug,
+To flash/debug the application click on the green bug icon on the top toolbar and select `Nucleo-L476RG Debug`.
+
+![Flash](images/eclipse_flash.png)
+
+If you cannot see it, select `Debug Configuration...` and it should be listed under `STM32 Cortex-M C/C++ Application` menu selection.
```

Comment:
@acwiek we would appreciate wrapping on 80 characters, this information should be in on-boarding documentation. What editor are you using?

@artur-rs @macpijan cc